var/spack/repos/builtin/packages/molgw/package.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from os import getenv
+from subprocess import PIPE, run
+
+from spack.package import *
+
+
+class Molgw(MakefilePackage):
+    """MOLGW is a quantum chemistry code that implements the many-body perturbation theory (MBPT)
+    to describe the excited electronic states in finite systems (atoms, molecules, clusters).
+    MOLGW implements the GW approximation for the self-energy (ionization and affinity)
+    and the Bethe-Salpeter equation for the optical excitations.
+    MOLGW also applies the real-time time-dependent density-functional theory (RT-TDDFT).
+    MOLGW employs standard Gaussian basis set.
+    """
+
+    homepage = "https://github.com/bruneval/molgw"
+    url = "https://github.com/bruneval/molgw/archive/v3.2.tar.gz"
+    git = "https://github.com/bruneval/molgw.git"
+
+    maintainers("bruneval")
+
+    # version("rolling-release", branch="master")
+    version("3.2", sha256="a3f9a99db52d95ce03bc3636b5999e6d92b503ec2f4afca33d030480c3e10242")
+
+    depends_on("blas")
+    depends_on("lapack")
+    depends_on("libxc@5:")
+    depends_on("libcint+pypzpx+coulomb_erf")
+
+    variant("openmp", default=False, description="Build with OpenMP support")
+
+    variant("scalapack", default=False, description="Build with ScaLAPACK support")
```

Comment:
(minor) Can we move variants above all the `depends_on` directives?

Reply:
Fixed!